File: src/Test/ListOpsFFICheatcode.go
package Test_ListOpsFFICheatcode


func RunListOpsFFICheatcode(limit float64) float64 {
	n := int(limit)
	sum := 0
	for i := 1; i <= n; i++ {
		if i%2 == 0 {
			sum += i
		}
	}
	return float64(sum)
}

File: src/Test/ListOpsFFICheatcode.java
public static final java.util.function.Function<Object, Object> runListOpsFFICheatcode = Bench.nativeBenchmark(input -> {
    int limit = ((Number) input).intValue();
    int sum = 0;
    for (int value = 1; value <= limit; value++) {
        if (value % 2 == 0) sum += value;
    }
    return sum;
});
